cleanlab/object_detection/filter.py

```diff
@@ -114,9 +120,10 @@ def find_label_issues(
 def _find_label_issues(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
-    *,
     return_indices_ranked_by_score: bool = True,
```

Comment:
Param `return_indices_ranked_by_score` and `scoring_method` should be after * as well